src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -181,6 +184,14 @@ export class KernelDebugAdapter implements DebugAdapter {
     dispose() {
         this.messageListener.forEach((ml) => ml.dispose());
         this.messageListener.clear();
+
+        // clean temp files
+        this.cellToFile.forEach((tempPath) => {
+            const norm = path.normalize(tempPath);
+            const dir = path.dirname(norm);
+            void this.fs.deleteLocalFile(norm);
```

Comment:
Do you need to try catch on any of these?

Reply:
good idea